pkgs/development/php-packages/php-cs-fixer/default.nix

```diff
@@ -1,28 +1,21 @@
-{ mkDerivation, fetchurl, makeWrapper, lib, php }:
+{ buildPhpProject, fetchFromGitHub, lib, php }@inputs:
 
-let
+buildPhpProject rec {
   pname = "php-cs-fixer";
-  version = "3.13.1";
-in
-mkDerivation {
-  inherit pname version;
+  version = "3.16.0";
 
-  src = fetchurl {
-    url = "https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v${version}/php-cs-fixer.phar";
-    sha256 = "4bQrCjuaWN4Dbs1tkk4m1WxSb510ue7G59HA+gQ52yk=";
+  src = fetchFromGitHub {
+    owner = "FriendsOfPHP";
+    repo = "PHP-CS-Fixer";
+    rev = "v${version}";
+    hash = "sha256-d8oypyaR8B7DHP92UtnRLuUv4SRKAOJ8cGDLxw3IlwM=";
   };
 
-  dontUnpack = true;
+  composerLock = ./composer.lock;
+  vendorHash = "sha256-+GcLij5r9Y/j87iMLckQPqa7VuqQpeYGSY74xcUZU1I=";
 
-  nativeBuildInputs = [ makeWrapper ];
-
-  installPhase = ''
-    runHook preInstall
-    mkdir -p $out/bin
-    install -D $src $out/libexec/php-cs-fixer/php-cs-fixer.phar
-    makeWrapper ${php}/bin/php $out/bin/php-cs-fixer \
-      --add-flags "$out/libexec/php-cs-fixer/php-cs-fixer.phar"
-    runHook postInstall
+  postInstall = ''
+     ln -s $out/share/php/${pname}/${pname} $out/bin/${pname}
```

Comment:
Should not this be done automatically based on the `bin` key in `composer.json`?

Reply:
Yes, this is something to be done.

Reply:
Done.